### ISSUE TITLE
No Games To Simulate

### ISSUE THREAD

**Issue by MunkeyShynes** [codecombat/codecombat#4090]
Not sure what's up with this. I've not been able to run simulations for over 24 hours now. Is anyone else simulating?

This is what I see in two different locations so I'm quite sure that it's not the connection on my end.  I've experienced this issue on 3 different computers at two different locations. One of the locations is a business with a 20 meg connection. All of them exhibit the same symptoms. This just started yesterday afternoon.

![nosimulate](https://cloud.githubusercontent.com/assets/19597966/21075874/2fcb1cc4-beeb-11e6-8757-fca3f294e2d3.JPG)

**Comment by vn971** [comment]
Yeah, I think it's a bug. Hope the team will fix that soon enough.

// I've put a new cool algo to play on "multiplayer-treasure-grove", it's even able to beat some of the TOP-10 guys with only the basic heroes, very much waiting for an update on the subject...))

**Comment by sderickson** [comment]
We were hit hard by [Hour of Code](https://code.org/learn) last week, particularly Friday, so we turned off simulations to take some of the load off. They should be back up soon.

**Comment by sderickson** [comment]
Reopening, renaming because [MunkeyShynes had a good point over here](https://github.com/codecombat/codecombat/issues/4083#issuecomment-266596352).
